.travis.yml

```diff
@@ -3,11 +3,8 @@ language: node_js
 node_js:
   - 6
 before_script:
-  - cd source && npm install && cd ..
-  - npm install -g cucumber@^1.0.0
-  - npm install -g gulp
-  - cp source/apickli/apickli-gherkin.js source/test/features/step_definitions/
-script:
   - cd source
-  - gulp lint
-  - cucumber-js test/ --tags @core && gulp test
+  - npm install
+script:
+  - node_modules/.bin/gulp lint
```

Comment:
does gulp lint only lint the code? or does it also run the tests?
we may want to include gulp test too